### PR TITLE
Removed the dedicated admin page

### PR DESCRIPTION
**Pull request by ryanlerch**
The admin page is now removed. the admin actions
now show up on the other pages if the user has admin
access.

### CODE REVIEW

File: fedora_elections/templates/_macros.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  <h3>{{title}}</h3>
+
+  <table class="table">
+    <tr>
+      <th class="stretch-cell">Election</th>
+      <th></th>
+      <th class="nowrap">Start date (UTC)</th>
+      <th class="nowrap">End date (UTC)</th>
+      {% if is_admin %}
+      {% endif %}
```
Comment (pypingou) [inline fedora_elections/templates/_macros.html]: Something got removed in between?

File: fedora_elections/templates/_macros.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          <a class="btn btn-sm btn-secondary m-l-2" href="{{ url_for('election_results',
+                  election_alias=election.alias) }}">
+          View Results
+          </a>
+          {% endif %}
+
+          {% if is_admin %}
+            <a class="btn btn-sm btn-danger" href="{{ url_for('admin_view_election',
+                                election_alias=election.alias) }}">
+              Edit
```
Comment (pypingou) [inline fedora_elections/templates/_macros.html]: Maybe more ``Review`` than ``Edit`` no?

**Comment by pypingou** [comment]
Couple of comments, the rest looks fine.

I see there are still a few page to adjust are you on it and shall I list them somewhere?

**Comment by pypingou** [comment]
:+1: for me, thanks! :)

**Comment by ryanlerch** [comment]
Fixed the two issues!

**Pull request merged**